creator-node/src/services/stateMachineManager/stateReconciliation/index.js

```diff
@@ -38,25 +58,36 @@ class StateReconciliationManager {
     })
 
     // Clear any old state if redis was running but the rest of the server restarted
-    await queue.obliterate({ force: true })
+    await reconciliationQueue.obliterate({ force: true })
+    await manualSyncQueue.obliterate({ force: true })
```

Comment:
we can remove the obliterate to keep `waiting` jobs between restarts. sorry to increase the scope here but it would be useful to change to clean() if you could confirm that clean() only removes `active` jobs and leaves `waiting` alone 🙏